sympy/printing/glsl.py

```diff
@@ -281,6 +281,12 @@ def _print_int(self, expr):
     def _print_Rational(self, expr):
         return "%s.0/%s.0" % (expr.p, expr.q)
 
+    def _print_Relational(self, expr):
+        lhs_code = self._print(expr.lhs)
+        rhs_code = self._print(expr.rhs)
+        op = expr.rel_op
+        return ("{0} {1} {2}").format(lhs_code, op, rhs_code)
```

Comment:
```suggestion
        return "{0} {1} {2}".format(lhs_code, op, rhs_code)
```
Can you remove these additional parentheses here and every below?